docs/modules/ROOT/pages/index.adoc

```diff
@@ -218,25 +218,28 @@ This specification defines a _<<C2PA assertion>>_ known as an *<<_identity_asser
 
 The *<<_identity_assertion,identity assertion>>* contains the following data fields:
 
-* `signer_payload` contains the set of data to be signed by this _<<_credential_holder,credential holder>>_. This structure is a CBOR key-value map that contains:
-** `referenced_assertions` _(required)_ contains a list of other assertions selected by the credential subject. This is referred to as `signer_payload.referenced_assertions` throughout this specification.
+* `signer_payload` contains the set of data to be signed by this _<<_credential_holder,credential holder>>_. 
 * `signature` contains the raw byte stream of the _<<_credential_holder,credential holder>>’s_ signature
-* `sig_type` defines the data type of signature
 * `pad1` and `pad2` are byte strings filled with binary `0x00` values used to fill space
 
-The content of `signer_payload` is a data structure which is presented to the _<<_credential_holder,credential holder>>_ for signature. The list of _<<_referenced_assertions,referenced assertions>>_ contained within `signer_payload` MUST include a _<<hard_binding,hard binding>>_ assertion as described in link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_hard_bindings++[Section 9.2, “Hard bindings,”] of the C2PA technical specification.
+The `signer_payload` is a structure of type `signer-payload-map` which signed by the _<<_credential_holder,credential holder>>_. It contains the following fields:
 
-IMPORTANT: A single signature from the _<<_credential_holder,credential holder>>_ covers the entire `signer_payload` data structure, including the _list_ of assertions. An implementation MUST NOT present each assertion separately for signature.
+* `sig_type` is a string identifying the data type of the signature.  Valid credential types and the corresponding `signature` data structures and `sig_type` values are defined in xref:_credentials_signatures_and_validation_methods[xrefstyle=full].
 
-For each assertion listed in the `signer_payload.referenced_assertions` field of the *<<_identity_assertion,identity assertion>>,* an assertion with the same `url`, `alg`, and `hash` values MUST also be listed in the `assertions` field of the _<<C2PA claim>>_ in which the *<<_identity_assertion,identity assertion>>* appears.
+* `referenced_assertions` is an array of `hashed-uri-map` structures with one entry for
+every assertion in the Claim, including the current assertion. While every 
+assertion must be referenced, the hash may be omitted for all but one assertion.
```

Comment:
(Later update): What I _think_ you're trying to do here looks similar to the `created_assertions` and `gathered_assertions` distinction that was introduced in the C2PA 2.0 specification. If I've understood that correctly, I'd like to explore the idea of making a similar distinction here rather than using the presence or absence of the `hash` field to imply responsibility.

Reply:
Is the intention to provide some protection against assertion stripping (even if that assertion came from a different source)?

Reply:
I think the attack may be assertion _addition_ if I read the linked issue and this correctly. As in, crudely illustrated:

- the C2PA claim generator sig is removed
- a new identity assertion and signature is added
- the new manifest is signed again, possibly by the same claim generator as the original

Reply:
This is exactly the issue I was trying to mitigate.  I started off with two "arrays" one for the assertions with the hashes, and one just containing the JUMBF URIs without the hashes.  I "simplified" this to hashed-URIs missing hashes, but don't have any particular attachment to this solution.

Reply:
re: @scouten-adobe - 

> The requirement that all assertions in the claim must be referenced creates a potential circular-reference problem should there be multiple authors

As @andyparsons divined, circular references are avoided by omitting the ID hash if the ID-assertion-hash is not yet known.  Once the earlier ID assertion is signed, the earlier-signer "loses control" of how the ID assertion is used (especially, if the c2pa claim-signature trust list contains bad actors.)  The ID assertion limits future/unwanted changes using the mechanisms described.

The proposal does have one limitation that we might want to talk about.  When the ID assertions are created, the JUMBF URI of all of the assertions in the claim (and most of the hashes) must be known.  This would be the case if the ID assertion signatures followed by the Claim Signature are all done one after another once "everything is known."  

However, if we think that the earlier ID assertion signer signs without knowing how their assertion will be incorporated by the next ID signer or the Claim signer, this won't work.  My thought was that this was best supported using Ingredients, but we could discuss.

Reply:
re @scouten-adobe 

> I am not sure why you're proposing to allow identity assertions to omit the hash reference to other assertions. I'll need to think through it more carefully, but this feels like a potential security hole.

Two cases:
1.  An ID assertion chooses to not include the hash because they don't want to "vouch for it."  Whether this is a security hole depends on your point of view.  If we want to include the hash without vouching for it, we could add a new field like gathered-assertions
2. The main use case was (a) the hash of the ID-assertion currently being signed, and (b) the hash of ID assertions not yet signed.  Ie both cases when the hash is not known yet.  

The "security hole" you identify is real and is not possible to fix completely.  It is mitigated by allowing an ID creator to optionally specify the public key of the signer for ID assertions when the hash is not yet known.